web/db/migrations/20260804030000_iroh_scoped_discovery_indexes/migration.sql

```
@@ -0,0 +1,7 @@
+CREATE INDEX "iroh_endpoint_bindings_active_pairable_mac_scope_idx"
+ON "iroh_endpoint_bindings" USING btree ("user_id", "tag", "id")
+WHERE "revoked_at" is null and "platform" = 'mac' and "pairing_enabled" = true;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)